src/dmd/traits.d

```diff
@@ -1945,6 +1945,39 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
         auto tup = new TupleExp(e.loc, exps);
         return tup.expressionSemantic(sc);
     }
+    if (e.ident == Id.getNamespaces)
+    {
+        auto o = (*e.args)[0];
+        auto s = getDsymbolWithoutExpCtx(o);
+        if(!s.cppnamespace)
+        {
+            auto exps = new Expressions(0);
+            auto tup = new TupleExp(e.loc, exps);
+            return tup.expressionSemantic(sc);
+        }
+        auto oexp = s.cppnamespace.exp;
+        import std.stdio;
+        import dmd.asttypename;
+        writeln("ASTtypeName ",astTypeName(oexp), " ", oexp.toString());
+        TupleExp tup;
+        if (auto otup = oexp.isTupleExp())
```

Comment:
This seems to not be triggered by the test 
```
extern(C++, mixin("AliasSeq!(`ns`, `nt`)")) struct GetNamespaceTest3 {}
```
not sure how to trigger.

Reply:
I just checked it seems only the last element get assigned to cppnamespace.exp not a tuple.
In `extern(C++, mixin("AliasSeq!(`ns`, `nt`)")) struct GetNamespaceTest3 {}` only the StringExp "nt" is there.

Reply:
Yes see the loop in the else branch. @Geod24 said this was because we are post semantic here.

Reply:
https://github.com/dlang/dmd/blob/master/src/dmd/attrib.d#L502-L511

Reply:
Yeah sorry I should've read the doc.

Reply:
Don't worry, I read it and still got confused by it.

Reply:
No sure how a tuple could be here, you could run expression semantic first to be really sure and simply delete this branch.
Moreover, semantic is run for each element anyway after the trait tuple creation.